src/main/java/org/zone/commands/structure/region/flags/messages/leaving/ZoneFlagLeavingRemoveCommand.java

```diff
@@ -28,15 +28,15 @@
     @Override
     public @NotNull List<CommandArgument<?>> getArguments() {
         return Arrays.asList(new ExactArgument("region"),
-                new ExactArgument("flag"),
-                ZONE,
-                new ExactArgument("leaving"),
-                new ExactArgument("remove"));
+                             new ExactArgument("flag"),
+                             ZONE,
```

Comment:
How come the extra tabs?

Reply:
what? oh the spaces? why not? XD I did it